Sources/SourceGraph/Elements/Declaration.swift

```diff
@@ -231,6 +231,7 @@ public final class Declaration {
     public var related: Set<Reference> = []
     public var isImplicit: Bool = false
     public var isObjcAccessible: Bool = false
+    public var referencedFiles: Set<SourceFile>
```

Comment:
What's the purpose of this property? I see that we insert items into it, but it doesn't appear to be used anywhere.